### DISCUSSION TITLE
PSA: Maintenance help needed!

### DISCUSSION THREAD

**Discussion by Metadorius** [Phobos-developers/Phobos#1262] [Announcements]
We, the Phobos developers, are currently in the situation where we **can't keep up with the amount of contributions received** (we currently have **over 80 open pull requests** aka pending contributions from the community programmers). Generally the main problem is that the contributions need to be *manually reviewed*, and our reviewing capacity is too low to keep up, so the contributions don't receive enough attention and many of them keep gathering dust, some of them __for years__ already. This, of course, is not optimal, so while we're working on making it easier to review contributions - we may need some help on that front.

So, in short:
1) if you are a programmer or a reverse engineer and have interest in helping - we would be happy if you get involved in reviewing current pending pull requests on feature design, implementation details, code style etc. (as always, PM me in Discord for a starter pack for YR reverse-engineering);
2) spread the word about us requiring more devs, especially if you know someone who is able to help and wants to help - please tell them about us.
